src/js/event-target.js

```diff
@@ -11,7 +11,7 @@ EventTarget.prototype.on = function(type, fn) {
   // Remove the addEventListener alias before calling Events.on
   // so we don't get into an infinite type loop
   let ael = this.addEventListener;
-  this.addEventListener = Function.prototype;
+  this.addEventListener = () => {};
```

Comment:
I don't think it matters here at all since we just want an empty function while `Events.on` runs so we don't get into an infinite loop, so, there's no expectations being subverted here.
As for elsewhere, it really depends on what you want the context of your function to be since in arrow functions `this` and `arguments` are never bound.